frame/nomination-pools/src/lib.rs

```diff
@@ -686,6 +794,82 @@ impl<T: Config> BondedPool<T> {
 			.saturating_sub(T::StakingInterface::active_stake(&account).unwrap_or_default())
 	}
 
+	/// Set the pool's commission.
+	fn set_commission_current(mut self, commission: &Perbill, payee: T::AccountId) -> Self {
+		// Force a commission of `None` if a 0% commission is provided.
+		// Note that a 0% commission will replace the whole tuple with None.
+		let new_current =
+			if commission > &Perbill::from_percent(0) { Some((*commission, payee)) } else { None };
```

Comment:
```suggestion
		let commission =
			if commission > &Perbill::from_percent(0) { Some((*commission, payee)) } else { None };
```

Reply:
I suggest re-assigning `commission` here because you probably never want to _accidentally_ use the old value anymore.